### PR TITLE
[Tests] Add unittest for ToolRunner

### PR DESCRIPTION
**Pull request by YongseopKim**
Let's add unittest for ToolRunner.

ONE-vscode-DCO-1.0-Signed-off-by: Yongseop Kim <yons.kim@samsung.com>

### CODE REVIEW

File: src/Tests/Project/ToolRunner.test.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import {MockJob} from '../MockJob';
+
+suite('Project', function() {
+  suite('ToolRunner', function() {
+    const logger = new Logger();
+    let job = new MockJob('mockup');
+    suite('#getOneccPath()', function() {
+      test('returns onecc path as string', function() {
+        let toolRunner = new ToolRunner(logger);
+        let actual = toolRunner.getOneccPath();  // string or undefined
```
Comment (hyunsik-yoon) [inline src/Tests/Project/ToolRunner.test.ts:32]: Shouldn't we also need to check something like the following?
```
assert(actual !== undefined);
```

File: src/Tests/Project/ToolRunner.test.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      });
+    });
+    suite('#getRunner()', function() {
+      test('returns runner as Promise<string>', function(done) {
+        let toolRunner = new ToolRunner(logger);
+        const oneccPath = toolRunner.getOneccPath();
+        // oneccPath could be string or undefined. Avoid compiling error
+        if (oneccPath === undefined) {
+          assert.fail('oneccPath should be string type');
+          return;  // for successful ts-compiling
```
Comment (mhs4670go) [inline src/Tests/Project/ToolRunner.test.ts:45]: I removed `return` statement and compiled successfully. Which error have you met?

Comment (YongseopKim) [reply src/Tests/Project/ToolRunner.test.ts:45]: Oh, now I know the only conditional expression `oneccPath === undefined` is necessary.

File: src/Tests/Project/ToolRunner.test.ts (hunk 1 above)
Comment (YongseopKim) [reply src/Tests/Project/ToolRunner.test.ts:32]: AFAIK, `===` checks object instance and its value so that we can skip it.

Comment (YongseopKim) [reply src/Tests/Project/ToolRunner.test.ts:32]: I mean that `actual === '/usr/share/one/bin/onecc'` checks `actual is not undefined` implicitly.

File: src/Tests/Project/ToolRunner.test.ts (hunk 2 above)
Comment (YongseopKim) [reply src/Tests/Project/ToolRunner.test.ts]: Fixed

File: src/Tests/Project/ToolRunner.test.ts (hunk 1 above)
Comment (YongseopKim) [reply src/Tests/Project/ToolRunner.test.ts:32]: Please note me if I'm wrong. (You know, I'm not good at ts/js.)

Comment (hyunsik-yoon) [reply src/Tests/Project/ToolRunner.test.ts:32]: I am also ts newbie. :-)

I checked the following and found that `?` does the magic.

```typescript
let bar = (n: number) => {
    if (n % 2 === 0) return undefined; 
    else return 'bar';
} 

let foo = bar(2);
console.log((foo === 'bar') || foo?.endsWith('bar'));   // "undefined" is printed
console.log(foo?.endsWith('bar') || (foo === 'bar'));   // "false" is printed

foo = bar(1);
console.log((foo === 'bar') || foo?.endsWith('bar'));   // 'true' is printed
```

**Review by hyunsik-yoon — APPROVED** [review]

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**